### ISSUE TITLE
[Bug]: Credit Card Prompt Fix didn't work

### ISSUE THREAD

**Issue by NichFuture** [eaaasun/CCStopper#42]
### What happened?

When I tried to run it CCStopper's Credit Card Prompt Fix, it ran but didn't actually fix anything Paywall

I ran Stop Adobe Processes beforehand which seemed to run fine but nothing changed in Adobe CC

**Comment by eaaasun** [comment]
What version are you running, and could you check your firewall settings to see if the rule is there? It should be prefixed with ```CCStopper-```

**Comment by hanyu-ayako** [comment]
I faced the same issue as well. Firewall has the rule there.
![image](https://user-images.githubusercontent.com/112416229/187248220-7d3e953e-0083-443d-bcee-ef7a745638ac.png)

**Comment by ItsProfessional** [comment]
It seems that Adobe has patched the method CCStopper used to stop the credit card popup. Firewall blocking `Adobe Desktop Service.exe` doesn't seem to stop the popup anymore.

**Comment by DrGaylordSteambathPhD** [comment]
either its only been patched in certain regions or your network is configured incorrectly. it's still working as im typing this.

**Comment by ItsProfessional** [comment]
> either its only been patched in certain regions or your network is configured incorrectly. it's still working as im typing this.

Comment has been updated.

**Comment by ItsProfessional** [comment]
^^ cleaned up misinformation in a few comments above

**Comment by AbsentForeskin** [comment]
Keep in mind that any other software managing your network security may interfere with rules set via _Windows Defender Firewall with Advanced Security_. McAfee is specifically & frequently known to cause problems bypassing the payment prompt. I’m unfamiliar with McAfee and have yet to investigate a workaround, but user Emette in the GenP Discord has posted a potential solution:
![image](https://user-images.githubusercontent.com/107275087/189403203-a49b558d-61e4-4821-accc-4482469d3dbf.jpeg)

**Comment by AbsentForeskin** [comment]
Additionally, I realize this sounds trivial, but double check your _Firewall & network protection_ pane in _Windows Security_ and verify that the firewall is enabled for your active connection. Many users with this issue have gotten mixed up with their domain, private, and public network configurations resulting in a misconfigured firewall, or have unknowingly disabled it entirely.

**Comment by eaaasun** [comment]
> ^^ cleaned up misinformation in a few comments above

the deleted comment: ```"either its only been patched in certain regions or your network is configured incorrectly. it's still working as im typing this."```

There could be regional differences surrounding popups like these so it isn't completely invalid. I can also vouch for its functionality as of now.

**Comment by momo1093324** [comment]
Today I use it successfully, I am on version 1.1.3
The antivirus software firewall you downloaded should be turned off "all", not just the adobe cc software (I use avast)
Then apply the CCStopper rule and you can succeed

**Comment by AbsentForeskin** [comment]
Thanks for your input.

For added information--placing the firewall rule is essentially only half of the process; firewall and network security also need to be configured so they are managed through Windows Defender because other security providers generally do not consider Defender’s rules in their own security policies. Disabling all network related security settings in Avast probably forced a fallback to Defender for firewall management (in turn enforcing the rule placed by CCStopper) which should be a functional solution for anyone experiencing this issue, however if it doesn't fallback automatically you may need to enable _Microsoft Defender Firewall_ listed under your active network in Windows Security's _Firewall & network protection pane_.

tl;dr - mixing security policies is a mess and requires clean-up before it works properly

**Comment by matheusblins** [comment]
Hey guys!

I'm posting the solution for my problem with the CCstopper v1.2.0, as i've seen many people talk about the same issue but no one suggested/tried this fix.

When i oppened my batch file and tried to run the first option (Stop Processes) it was flashing the error 'no digital signature' (as it was very fast, i didn't paid attention to that in the firsts attempts). So i just unblocked all the files in the script folder (right click > properties > check the unblock box and hit apply), and now everything is working fine, no need to disable firewall/antivirus.

**Comment by eaaasun** [comment]
> Hey guys!
> 
> I'm posting the solution for my problem with the CCstopper v1.2.0, as i've seen many people talk about the same issue but no one suggested/tried this fix.
> 
> When i oppened my batch file and tried to run the first option (Stop Processes) it was flashing the error 'no digital signature' (as it was very fast, i didn't paid attention to that in the firsts attempts). So i just unblocked all the files in the script folder (right click > properties > check the unblock box and hit apply), and now everything is working fine, no need to disable firewall/antivirus.

Just updated with [v1.2.0-pre.2](https://github.com/eaaasun/CCStopper/releases/tag/v1.2.0-pre.2), please give that a try and see if you still need to unblock the file.

**Comment by FulDreadZ** [comment]
ACC stopped loading then it popped up telling me it was damaged and it basically redownlaoded itself now im all scrambled up on the steps, I tried Ending all the tasks and I turned off my Norton Firewall and Tried CCstopper and the rules are up and everything but the Credit card trial still pops up.

**Comment by eaaasun** [comment]
> ACC stopped loading then it popped up telling me it was damaged and it basically redownlaoded itself now im all scrambled up on the steps, I tried Ending all the tasks and I turned off my Norton Firewall and Tried CCstopper and the rules are up and everything but the Credit card trial still pops up.

Did you get an error something along the lines of "you may not have full access to Adobe products and services?" 

You can safely ignore that error if so. Adobe makes a Creative Cloud cleaner tool, I suggest you run that and reinstall everything.

**Comment by FulDreadZ** [comment]
It seems that Trial Banner on CCStopper does not seem to work for me, When I click it, it just goes back to the original screen.

**Comment by FulDreadZ** [comment]
> > ACC stopped loading then it popped up telling me it was damaged and it basically redownlaoded itself now im all scrambled up on the steps, I tried Ending all the tasks and I turned off my Norton Firewall and Tried CCstopper and the rules are up and everything but the Credit card trial still pops up.
> 
> Did you get an error something along the lines of "you may not have full access to Adobe products and services?"
> 
> You can safely ignore that error if so. Adobe makes a Creative Cloud cleaner tool, I suggest you run that and reinstall everything.

Will do

**Comment by eaaasun** [comment]
> It seems that Trial Banner on CCStopper does not seem to work for me, When I click it, it just goes back to the original screen.

Yeah, that's why it's not in v1.2.0-pre.2 lol

**Comment by eaaasun** [comment]
> yes and Will do. By reinstalling everything, do you mean the ACC apps and ACC or that AND the GENP and CCstopper as well?

I do believe it's just the Creative Cloud app, not any of the programs (Photoshop/Premiere/etc.)

No need to reinstall GenP or CCStopper unless you modified one of them.

**Comment by FulDreadZ** [comment]
> Owner

I'm not sure how to run the cleaner tool but I tried uninstalling creative cloud through windows, and it would give me two options; <Uninstall> <Repair>. However, it would not allow me to reinstall it because I have photoshop.

**Comment by FulDreadZ** [comment]
> > yes and Will do. By reinstalling everything, do you mean the ACC apps and ACC or that AND the GENP and CCstopper as well?
> 
> I do believe it's just the Creative Cloud app, not any of the programs (Photoshop/Premiere/etc.)
> 
> No need to reinstall GenP or CCStopper unless you modified one of them.

I have reinstalled CC and did the steps from Reddit to crack it but now it just sits on the loading screen.

**Comment by eaaasun** [comment]
> I have reinstalled CC and did the steps from Reddit to crack it but now it just sits on the loading screen.

Stop all adobe processes, then check the firewall/hosts file and remove entries made by CCStopper. CC should work now.

**Comment by RecursiveRegistrations** [comment]
Considering the popularity of this tool, I suppose it's unlikely, but I'll ask anyway: does applying the Credit Card Prompt fix cause anything else to break? I would think that if you block the desktop service from connecting to the internet, it would break things like file syncing and even in general the ability to download updates.

**Comment by afgks** [comment]
I have this problem as well. I'm on the latest version, and the firewall rules are created by the CCStopper successfully, but nothing happened to the credit card prompt screen. I have used the Cloud Cleaner Tool to clean everything before I started the installation process.
